### PR TITLE
Powershell script feature parity

### PR DESCRIPTION
**Pull request by ceruulean**
Hey, wanted to use this on windows. The md2short.ps1 is close to the bash version.

I did add a `test/` folder, was planning to do md2long version. Let me know if you want to remove it.

### CODE REVIEW

**Comment by davidlday** [comment]
Thanks for contributing! These are some welcome changes, as I haven't even thought about using these templates on Windows in a long time. Leave the `test/` folder. Whatever tests you can add are greatly appreciated! I'll add some GH actions to run them for later PRs.

**Comment by davidlday** [comment]
@ceruulean - Is this PR ready for review / merge? When I read your initial comment, I read it as you intended to add updates to `md2long` as part of this PR, but now I'm thinking you might do it separately. Either is fine, I just don't want this PR lingering if you're done with it.

**Comment by ceruulean** [comment]
@davidlday Sure, added a `md2long.ps1` file. Sorry for the weird commits, I was working on another account

File: README.md [hunk 1]
```diff
@@ -58,6 +58,14 @@ contact_phone: "(212) 555-1212"
 contact_email: "format@shunn.net"
 ```
 
+Optional properties for the frontpage:
```
Comment (davidlday) [inline README.md:61]: I don't see where these optional fields are used. Did you mean to implement them in the template(s)? If this is partial work, please take them out of the README and open an issue. We can incorporate them in a separate PR.

**Review by davidlday — CHANGES_REQUESTED** [review]
One change requested in the README regarding `Optional properties for the frontpage:`. The README also needs to be updated to reflect the new pwsh versions, but I can circle back and take care of those changes when I add in test automation.

File: README.md (hunk 1 above)
Comment (ceruulean) [reply README.md:61]: They're within the template.docx files
image

Oh, looks like there's a `date` too

**Result:**
image

Comment (davidlday) [reply README.md:61]: Oh! Yes, you're right. My mistake. I was thinking you were trying to add more custom metadata like I had to do for short_title, contact_name, etc. Yes, `subtitle`, `date`, and `abstract` are valid options and should be documented like you did. Thank you!

**Review by davidlday — APPROVED** [review]

**Pull request merged**